src/targets/node/program.ts

```diff
@@ -46,6 +46,28 @@ export class SubprocessProgram implements IProgram {
   }
 }
 
+/**
+ * A no-op program that never stops until stop() is called. Currently, we use
```

Comment:
Is this what we did in v1 as well?

Reply:
v1 actually does terminate it somehow. I had a mental bullet point, which is now a physical bullet point on https://github.com/microsoft/vscode-pwa/issues/58, to look at this some more.